docs/sections/user_guide/yaml/components/fv3.rst

```diff
@@ -119,50 +74,7 @@ Accepted values are ``global`` and ``regional``.
 execution:
 ^^^^^^^^^^
 
-batchargs:
-""""""""""
-
-These entries map to job-scheduler directives sent to e.g. Slurm when a batch job is submitted via the ``--batch`` CLI switch or the ``batch=True`` API argument. The only **required** entry is ``walltime``.
-
-Shorthand names are provided for certain directives for each scheduler, and can be specified as-so along with appropriate values. Recognized names for each scheduler are:
-
-* LSF: ``jobname``, ``memory``, ``nodes``, ``queue``, ``shell``, ``stdout``, ``tasks_per_node``, ``threads``, ``walltime``
-* PBS: ``debug``, ``jobname``, ``memory``, ``nodes``, ``queue``, ``shell``, ``stdout``, ``tasks_per_node``, ``threads``, ``walltime``
-* Slurm: ``cores``, ``exclusive``, ``export``, ``jobname``, ``memory``, ``nodes``, ``partition``, ``queue``, ``rundir``, ``stderr``, ``stdout``, ``tasks_per_node``, ``threads``, ``walltime``
-
-Other, arbitrary directive key-value pairs can be provided exactly as they should appear in the batch runscript. For example
-
-.. code-block:: yaml
-
-   --nice: 100
-
-could be specified to have the Slurm directive
-
-.. code-block: text
-
-   #SBATCH --nice=100
-
-included in the batch runscript.
-
-executable:
-"""""""""""
-
-The name of or path to the FV3 executable binary.
-
-mpiargs:
-""""""""
-
-An **array** of string arguments that should follow the MPI launch program (``mpiexec``, ``srun``, et al.) on the command line.
-
-mpicmd:
-"""""""
-
-The MPI launch program (``mpiexec``, ``srun``, et al.)
-
-threads:
-""""""""
-
-The number of OpenMP threads to use when running FV3.
+See :ref:`here <execution_yaml>` for details.
 
 field_table:
 ^^^^^^^^^^^^
```

Comment:
The code blocks below on lines 89 and 95 need a second colon. The files_to_copy section in the rendered version is missing some text.

Reply:
Added, thanks!